### ISSUE TITLE
Some Strange Output

### ISSUE THREAD

**Issue by lailvlong** [sinAshish/LGM-Net-Pytorch#5]
Hello, thank you very much for sharing your codes! When i run the codes on CIFAR-100, under 5-way, 1-shot setting, the output (preds) of the network is strange. As we can see from the figure, the prediction of every target image is [0.0006, 0.0006, 0.0006, 0.0006, 0.0006], whose sum should be 1. And during the training, this happen at every step, which makes the loss is always 1.60944.  Have you met this issue before? Do you know why this happend?  Look forward to your favourable reply. Thank you! 

![1582340114(1)](https://user-images.githubusercontent.com/29424397/75085426-183d9100-5564-11ea-89ee-0cfb99a3ccfc.jpg)
